tests/hpc/slurm_master_db.pm

```diff
@@ -137,7 +137,73 @@ sub test_06 {
 }
 
 sub test_07 {
-    ##TODO: Accounting check
+    my $name        = 'Slurm accounting';
+    my $description = 'Basic check for slurm accounting cmd';
+    my $result      = 0;
+    my %users       = (
+        'user_1' => 'Sebastian',
+        'user_2' => 'Egbert',
+        'user_3' => 'Christina',
+        'user_4' => 'Greg',
+    );
+
+    ##Add users TODO: surely this should be abstracted
+    script_run("useradd $users{user_1}");
+    script_run("useradd $users{user_2}");
+    script_run("useradd $users{user_3}");
+    script_run("useradd $users{user_4}");
+
+    script_run('sacctmgr -i add cluster linux');
+    my $cluster = script_output('sacctmgr -n -p list cluster');
+
+    if (index($cluster, 'linux') == -1) {
+        #cluster not successfully added
+        $result = 1;
+        goto FAIL;
+    }
+
+    ### Create accounts in org=UNI_X
+    script_run("sacctmgr -i add account UNI_X_IT Description=\"IT at UNI_X\" Organization=UNI_X");
+    script_run("sacctmgr -i add account UNI_X_Math Description=\"Math at ORG_X\" Organization=UNI_X");
+    #Add users associated with account in org=UNI_X
+    script_run("sacctmgr -i create user name=$users{user_1} DefaultAccount=UNI_X_Math");
+    script_run("sacctmgr -i create user name=Jose DefaultAccount=UNI_X_Math");
+    script_run("sacctmgr -i create user name=$users{user_2} DefaultAccount=UNI_X_IT");
+    script_run("sacctmgr -i create user name=Christian DefaultAccount=UNI_X_IT");
+
+    ### Create accounts in org=UNI_Y
+    script_run("sacctmgr -i add account UNI_Y_Physics Description=\"UNI_Y\" Organization=UNI_Y");
+    script_run("sacctmgr -i add account UNI_Y_Biology Description=\"UNI_Y\" Organization=UNI_Y");
+    #Add users associated with account in org=UNI_Y
+    script_run("sacctmgr -i create user name=Joe DefaultAccount=UNI_Y_Physics");
+    script_run("sacctmgr -i create user name=Noah DefaultAccount=UNI_Y_Biology");
+    script_run("sacctmgr -i create user name=$users{user_4} DefaultAccount=UNI_Y_Physics");
+    script_run("sacctmgr -i create user name=$users{user_3} DefaultAccount=UNI_Y_Biology");
+
+    script_run('sacctmgr show account');
```

Comment:
what about record_info to get this information quicker without opening the log?

Reply:
make sense